### PR TITLE
white anchor text in primary buttons

### PR DESCRIPTION
**Pull request by aimichal**
This makes anchor text white when used in a primary button.

Before:

image

After:

image

This fixes https://github.com/allenai/varnish/issues/286

### CODE REVIEW

File: components/button.tsx [hunk 1]
```diff
@@ -168,5 +168,9 @@ export const Button = styled(({ variant, ...props }) => <AntButton {...props} />
                     )};
             }
         }
+
+        a {
+            ${props => (props.type === 'primary' ? `&, &:hover { color: #ffffff; }` : null)}
```
Comment (jonborchardt) [inline components/button.tsx:173]: we dont want white
we want the varnish color .... parnish.palette.text.contrast (which is white)

Comment (aimichal) [reply components/button.tsx]: 👍

**Comment by aimichal** [comment]
> Do we set an `:active` or `:visited` color on links that we need to override too?

When I hover I see:

> image

When I click and hold it looks like that too.

After visiting, and returning to the button page, the button looks like it did before I clicked.

Is that what you were asking?

**Comment by codeviking** [comment]
Yup, thanks for checking!

**Pull request merged**